jazz-build-module/apigee/Build/prepareProxyPom.xml

```diff
@@ -0,0 +1,191 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>apigee</groupId>
+	<artifactId>clone-new-template-pom</artifactId>
```

Comment:
Can we use better artifactId? `clone-new-template-pom` looks like a copy+paste miss :)